tests/unit/dom.ts

```diff
@@ -237,5 +237,169 @@ registerSuite({
 			assert.strictEqual(parent.children.length, 0);
 			assert.isNull(node.parentNode);
 		}
-	}
+	},
+
+	'CSS manipulation': (function () {
+		function cssSuite(element: Element): {} {
+			let targetElement = <any> element;
+			return {
+				addClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'add single class'() {
+						assert.notInclude(targetElement.className, 'test');
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+					},
+
+					'add multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.addClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.addClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
```

Comment:
The `classList` API behaves different for an explicitly null class vs. no specified class. I updated the descriptions to be clearer, because your point about lack of clarity still stands.